README.md

```diff
@@ -48,3 +48,24 @@ editor.addControls([draw, cad]);
 ## Development
 - Build: `npm run build`
 - Create doc: `npm run-script doc`
+
+### Versions and Changelog
+This repo uses [standard-version](https://github.com/conventional-changelog/standard-version/) for release versioning and changelog management. Therefore updates should be committed using [conventional commit](https://www.conventionalcommits.org/en/v1.0.0/) messages:
+
+```
+<type>[optional scope]: <description>
+
+[optional body]
+
+[optional footer(s)]
+```
+
+The commit contains the following structural elements, to communicate intent to the consumers of your library:
+
+1. fix: a commit of the type fix patches a bug in your codebase (this correlates with PATCH in semantic versioning).
+2. feat: a commit of the type feat introduces a new feature to the codebase (this correlates with MINOR in semantic versioning).
+3. BREAKING CHANGE: a commit that has a footer BREAKING CHANGE:, or appends a ! after the type/scope, introduces a breaking API change (correlating with MAJOR in semantic versioning). A BREAKING CHANGE can be part of commits of any type.
+4. types other than fix: and feat: are allowed, for example @commitlint/config-conventional (based on the the Angular convention) recommends build:, chore:, ci:, docs:, style:, refactor:, perf:, test:, and others.
```

Comment:
@danji90 here commitlint is mentionned

Reply:
I already implemented it locally, but am waiting for @mhaertwig or @friedjoff to comment in the ticket about if we are ok with using yarn only